src/api/firebase.js

```diff
@@ -0,0 +1,83 @@
+import firebase from "firebase/app";
+import "firebase/auth";
+
+import ERROR from "../constants/error";
+import MESSAGE from "../constants/messages";
```

Comment:
단수, 복수 통일하면 좋을 것 같습니다.

```suggestion
import ERRORS from "../constants/errors";
import MESSAGES from "../constants/messages";
```